mqtt/mqtt-bridge/src/config_update.rs

```diff
@@ -136,7 +136,6 @@ impl BridgeUpdate {
         }
     }
 
-    // TODO update should have name
```

Comment:
`name` is not a part of twin config anymore. So I guess we either need to add that to the config, or remove it from rust code.

Reply:
Oh, I see. We already have `endpoint()` below. Missed that... I think I'll try to remove `name`.